Header header
float32 LatDec
float32 LonDec
float64 altitude
float64 fix_quality
float64 easting
float64 northing
float64 zone_number
string zone_letter